src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Image/Image.ts

```diff
@@ -122,20 +136,41 @@ export var showImageDirective = (
             cssClass: "@",
             alt: "@?",
             format: "@?", // defaults to "detail"
-            imageMetadataNick: "@?" // defaults to SIImageMetadata.nick
+            imageMetadataNick: "@?", // defaults to SIImageMetadata.nick
+            fallbackUrl: "@?", // defaults to "/static/fallback_$format.jpg";
+            isImageMissing: "=?"
         },
         link: (scope) => {
-            if ( ! scope.imageMetadataNick) { scope.imageMetadataNick = SIImageMetadata.nick; }
-            if ( ! scope.format) { scope.format = "detail"; }
-
-            var fallbackUrl = "/static/fallback_" + scope.format + ".jpg";
+            var imageMetadataNick = () =>
+                scope.imageMetadataNick ? scope.imageMetadataNick : SIImageMetadata.nick;
+            var format = () => scope.format ? scope.format : "detail";
```

Comment:
The problem I found is that if angular updates the data-format property at any point later, the default set this way is overridden.

So the scope.format || 'detail' way to set a default only works if you do that in a template, but not in the link function that is not re-run when this happens.